JustSaying/AwsTools/MessageHandling/SqsNotificationListener.cs

```diff
@@ -210,6 +217,38 @@ internal async Task ListenLoopAsync(CancellationToken ct)
             }
         }
 
+        /// <summary>
+        /// Creates a task that extends the message visibility of the batch on a cadence of half the timeout duration, until all of the messages have been processed.
+        /// </summary>
+        /// <param name="inflightTracker"></param>
+        /// <param name="cancellationToken"></param>
+        private void StartBatchHeartbeat(MessageBatchInflightTracker inflightTracker, CancellationToken cancellationToken)
+        {
+            _ = Task.Run(async () =>
```

Comment:
I always feel like passing a cancellation to `Task.Run` is overkill the process it's running is cancellable 🤷‍♂️